### PR TITLE
Add method for overriding some cli commands

### LINKED ISSUE #99: openhab-cli: tail vs. multitail, function override
https://github.com/openhab/openhab-linuxpkg/blob/d001188d66cc10ff711acd417f02c85c3ee3d2de/resources/usr/bin/openhab-cli#L122-L124

@BClark09 openHABian now includes a multitail color scheme: https://github.com/openhab/openhabian/issues/196

How do you feel about multitail vs tail? The two panes and colors of multitail are clearly nice but I sometimes miss the option to scroll back up in tail (in my console). Which do you favor?
(If you want to see the difference, execute `openhablog`, `openhablog2` and  `openhablog3` on a fresh openHABian system.)

Going one step further, I wonder if it would be beneficial to be able to tune the behavior of the stiff distribution script `openhab-cli` on an openHABian system or potentially other systems. I'd fancy the option to modify how and what `openhab-cli` presents when called with the `showlogs` argument. How do you feel about the following proposal: IF certain alias exists THEN execute alias ELSE execute default routine. We could add such a behavior to all subroutines (even though I'd be interested in the log one).

What do you think? Did I miss anything? A potential security risk maybe?

### PR DESCRIPTION
**Pull request by BClark09**
Closes #99 

Signed-off-by: Ben Clark <ben@benjyc.uk>

### CODE REVIEW

**Comment by ThomDietrich** [comment]
@BClark09 everything looks perfect as always, from functions down to comments.

Regarding the location: At first I thought scripts are not fitting for `/etc` but at second thought, I don't know why the wouldn't. The only kinda stupid argument against it would be tab-completion on the console (`/etc/openh<TAB>`) 😊 

LGTM, 🥇 thanks!

File: resources/usr/bin/openhab-cli [hunk 1]
```diff
@@ -34,13 +34,23 @@ checkRoot() {
...
+
 # Load the default directory paths and any possible custom ones set
 if [ -r /etc/profile.d/openhab2.sh ]; then
   . /etc/profile.d/openhab2.sh
 elif [ -r /etc/default/openhab2 ]; then
   . /etc/default/openhab2
 fi
 
+# Load the override scripts to replace full comands
+if [ -r /etc/openhab-cli/command-overrides.sh ]; then
```
Comment (ThomDietrich) [inline resources/usr/bin/openhab-cli]: `-r` NICE!

Maybe should mention this in the comment though?

**Comment by BClark09** [comment]
Thanks a lot @ThomDietrich! Now that we have a folder for customisation, it will be easier to implement extra overrides like this. Just let me know if you've got any ideas about it! :)

**Comment by ThomDietrich** [comment]
Thanks!

**Pull request merged**